### ISSUE TITLE
monitoring training and cross-validation loss values

### ISSUE THREAD

**Issue by Mkarami3** [koyeongmin/PINet#35]
Hello,
Many thanks for providing your code for free.
I am training the model on my own dataset. I wonder how can I monitor the loss values for training and cross-validation?
Mohammad

**Comment by koyeongmin** [comment]
Because TuSimple dataset has only about 3000 frames for training,  the code evaluated the performance about test set every epoch.

**Comment by NEFUtraffica** [comment]
> 您好，非常感谢您免费提供您的代码。我正在自己的数据集上训练模型。我想知道如何监控训练和交叉验证的损失值？ 穆罕默德

Hello, I also want to see the loss function change during training, have you succeeded, can I communicate with you
